pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -70,6 +70,10 @@ type DependenciesDistributor struct {
 	RESTMapper          meta.RESTMapper
 	ResourceInterpreter resourceinterpreter.ResourceInterpreter
 
+	// ConcurrentResourceTemplateSyncs is the number of resource templates that are allowed to sync concurrently.
+	// Larger number means responsive resource template syncing but more CPU(and network) load.
```

Comment:
A larger number means syncing resource templates faster but more CPU(and network) load